crm_lead_date_duration/views/crm_lead_views.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_crm_lead_form_inherit" model="ir.ui.view">
+        <field name="name">crm_lead_form (in CRM Lead Date Duration)</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group//field[@name='date_deadline']" position="before">
+                <field name="event_date" />
+                <label for="duration"/>
+                <div>
+                    <field name="duration" widget="float_time" string="Duration" />
```

Comment:
hours label splits in two lines:

![imagen](https://user-images.githubusercontent.com/12749832/232804812-277d48b8-ccd5-49e4-a803-db480829c891.png)


This is fixed in this way:
```suggestion
                    <field name="duration" widget="float_time" string="Duration" class="oe_inline"/>
```